.pytest_cache/v/cache/lastfailed

```diff
@@ -1,31 +1,13 @@
 {
```

Comment:
Please remove this file from git. You can add `.pytest_cache` to .gitignore file.